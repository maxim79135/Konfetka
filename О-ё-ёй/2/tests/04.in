30
98163 86892 -10033 -18806 -60979 45601 -14553 53676 29561 19193 -13969 -75433 -8578 74485 15256 90716 21067 23625 -32057 82877 66310 49760 -17988 -46019 -47719 -37948 -84942 38019 23035 -76086 
