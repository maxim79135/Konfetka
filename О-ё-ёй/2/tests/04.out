98163 90716 86892 82877 74485 66310 53676 49760 45601 38019 29561 23625 23035 21067 19193 15256 -8578 -10033 -13969 -14553 -17988 -18806 -32057 -37948 -46019 -47719 -60979 -75433 -76086 -84942 